components/dashboards/org.wso2.carbon.dashboards.api/src/main/java/org/wso2/carbon/dashboards/api/internal/DashboardRestApi.java

```diff
@@ -182,4 +186,66 @@ public Response delete(@PathParam("id") String id) {
             return Response.serverError().entity("Cannot delete dashboard '" + id + "'.").build();
         }
     }
+
+    @GET
+    @Path("/roles")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getRoles() {
+        try {
+            List<Role> allRoles = dashboardDataProvider.getAllRoles();
+            return Response.ok()
+                    .entity(allRoles)
+                    .type(MediaType.APPLICATION_JSON)
```

Comment:
Since we have added the `@Produces(MediaType.APPLICATION_JSON)` at method level, no need to set the media type here.
Please remove here & other places.

Reply:
Ack.